includes/class-events-store.php

```diff
@@ -91,25 +58,14 @@ protected function class_init() {
 	 */
 	public static function is_installed() {
 		$db_version = (int) get_option( self::DB_VERSION_OPTION );
-
 		return version_compare( $db_version, 0, '>' );
 	}
 
 	/**
 	 * Build appropriate table name for this install
 	 */
 	public function get_table_name() {
-		global $wpdb;
-
-		return $wpdb->prefix . self::TABLE_SUFFIX;
-	}
-
-	/**
-	 * Set initial options that control plugin's behaviour
-	 */
-	protected function prime_options() {
-		// Prime DB option.
-		add_option( self::DB_VERSION_OPTION, 0, null, false );
+		return self::table_name();
```

Comment:
Why this change? I am not a big fan of static functions. but I get that you now need it in the other new static functions :worried:

Reply:
This specific change was just to help remove some of the duplication.